Statut_HSP.tex

```diff
@@ -168,19 +174,47 @@ \section{Członkowie Stowarzyszenia}
   regularnego opłacania składek.
 \end{enumerate}
 
-\myparagraph{§ 14}
+\myparagraph{§ 12.NZ.1}
+\begin{enumerate}
+\item
+  Prawa i obowiązki każdego członka nadzwyczajnego z osobna definiuje zarząd.
+\item
+  Prawa i obowiązki członka nadzwyczajnego mogą być zmienione na jego wniosek po zatwierdzeniu przez zarząd.
+\item
+  Prawa członka nadzwyczajnego muszą zawierać się w zbiorze praw członka zwyczajnego.
+\item
+  Członek nadzwyczajny musi mieć obowiązek regularnego opłacania składek członkowskich.
+\item
+  Członek nadzwyczajny musi mieć obowiązek przestrzegania statutu i uchwał władz stowarzyszenia.
+\item 
+  Członek nadzwyczajny musi nie posiadać biernego oraz czynnego prawa wyborczego.
+\item
+  Członek zwykły może w każdym momencie zostać członkiem nadzwyczajnym informując w formie pisemnej zarząd.
+\end{enumerate}
+
+\myparagraph{§ 12.W.1}
 \begin{enumerate}
 \item
-  Członkowie wspierający i honorowi nie posiadają biernego oraz czynnego prawa wyborczego, mogą jednak brać udział z głosem doradczym w statutowych władzach Stowarzyszenia, poza tym posiadają takie prawa jak członkowie zwyczajni.
+  Członkowie wspierający nie posiadają biernego oraz czynnego prawa wyborczego, mogą jednak brać udział z głosem doradczym w statutowych władzach Stowarzyszenia, poza tym posiadają takie prawa jak członkowie zwyczajni.
```

Comment:
Nie wiem czy "w statutowych władzach Stowarzyszenia" nie jest zbyt szerokie, ponieważ oprócz Walnego (najwyższej formy władzy) dotyczy też Zarządu i Komisji Rewizyjnej. To wtedy jest jakby członkiem honorowym tych gremiów? Możesz być, ale nie możesz głosować - bezużyteczne.

IMO expicite powinno być zabrionione uczestnictwo w władzach. Głos doradczy (tudzież "opiniująco-doradczy" jak to mają w Samorządzie Studentów PG) jest ważny tylko na Walnym i jest tak jakby tylko zaproszeniem do uczestnictwa w Walnym.

Reply:
Tzn to mieliśmy już w statucie przedtem - nie chcę tego zmieniać, by coś dziwnego nie wyszło.
I tak to nie jest nadużywane nigdzie.